spec/Entity/CreditMemoSpec.php

```diff
@@ -24,7 +24,7 @@ function let(): void
             1000,
             'USD',
             'en_US',
-            new CreditMemoChannel('WEB-US', 'United States', 'Linen'),
+            new Channel(),
```

Comment:
```suggestion
            $channel,
```
and put `ChannelInterface $channel` in `let()` method